src/aliceVision/localization/optimization.cpp

```diff
@@ -327,6 +324,13 @@ bool refineSequence(std::vector<LocalizationResult>& vec_localizationResult,
             }
             currResult.updateIntrinsics(params);
 
+            auto curDistortion = currResult.getIntrinsics().getDistortion();
```

Comment:
```suggestion
            auto currDistortion = currResult.getIntrinsics().getDistortion();
```
For coherence, since the localization result is "currResult".